frontend/src/config.ts

```diff
@@ -49,6 +52,8 @@ export function getConfig(): Config {
         },
         api: {
             baseUrl: process.env.NEXT_PUBLIC_API_BASE_URL,
+            roundRobinUrl: process.env.NEXT_ROUND_ROBIN_API,
+            roundRobinAuthToken: process.env.NEXT_ROUND_ROBIN_AUTH,
```

Comment:
I'm a bit confused about the purpose of the auth token. Isn't this public data? Why are we protecting this with a token?

Reply:
try this shouldn't be here